### PR TITLE
better css for upcoming events paragraph

### PR DESCRIPTION
**Pull request by retif**

### CODE REVIEW

File: themes/openy_themes/openy_carnation/openy_carnation.theme [hunk 1]
```diff
@@ -320,6 +320,15 @@ function openy_carnation_theme_suggestions_page_alter(array &$suggestions, array
   }
 }
 
+/**
+ * Implements hook_theme_suggestions_HOOK_alter().
+ */
+function openy_carnation_theme_suggestions_views_view_alter(array &$suggestions, array $variables) {
+  if($variables['view']->id() === 'latest_event_posts') {
+    $suggestions[] = 'views-view__latest_event_posts';
```
Comment (anpolimus) [inline themes/openy_themes/openy_carnation/openy_carnation.theme]: I think, in this plave default view suggestions should work
views-view--latest_event_posts.html.twig

Comment (retif) [reply themes/openy_themes/openy_carnation/openy_carnation.theme]: Theme debug shows only views-view.html.twig suggestion there, so i added this one

Comment (anpolimus) [reply themes/openy_themes/openy_carnation/openy_carnation.theme]: There is a bug in drupal core that is not showing theme suggestions at views section

Comment (anpolimus) [reply themes/openy_themes/openy_carnation/openy_carnation.theme]: https://api.drupal.org/api/drupal/core%21modules%21views%21views.theme.inc/group/views_templates/8.2.x
Try read this and apply working suggestion

**Pull request merged**